### PR TITLE
Adds feedback button

### PR DESCRIPTION
**Pull request by ChristopherPark3**
# Description & Technical Solution
- Implements in `header.tsx`
- Integrates Featurebase SDK 
  - Both widget and auth (https://help.featurebase.app/en/articles/4711831-feedback-widget)
- Interfaces with Featurebase, passing Sorbet signed in users for auth (https://help.featurebase.app/en/articles/6151321-sdk-auto-authentication-and-data-sync-setup)

# Video

https://github.com/user-attachments/assets/05a79e11-968d-43f0-98ad-0b9db2d7f706

### CODE REVIEW

File: src/components/profile/featurebase-widget.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+'use client'; // NextJS 13 requires this. Remove if you are using NextJS 12 or lower
+import Script from 'next/script';
+import { useEffect } from 'react';
+
+import { Button } from '@/components/ui/button';
+import { useAuth } from '@/hooks';
+
+const FeaturebaseWidget = () => {
```
Comment (Cutaiar) [inline src/components/profile/featurebase-widget.tsx:8]: Want to link to the docs where you got this code?

File: src/components/header/header.tsx [hunk 2]
```diff
@@ -33,6 +34,16 @@ export const Header = () => {
         {user && (
           <div className='flex items-center justify-end gap-4'>
             <div className='align-center flex flex-row items-center gap-2'>
```
Comment (Cutaiar) [inline src/components/header/header.tsx:36]: Remove comment if not using

File: src/components/profile/featurebase-widget.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
+'use client'; // NextJS 13 requires this. Remove if you are using NextJS 12 or lower
```
Comment (Cutaiar) [inline src/components/profile/featurebase-widget.tsx:1]: You can remove this comment, yeah?

**Review by Cutaiar — APPROVED** [review]

File: src/components/profile/featurebase-widget.tsx (hunk 3 above)
Comment (ChristopherPark3) [reply src/components/profile/featurebase-widget.tsx:1]: Yea i'll remove all comments!

**Pull request merged**